code/modules/mob/living/status_procs.dm

```diff
@@ -66,10 +66,17 @@
 	if(((status_flags & CANKNOCKDOWN) && !HAS_TRAIT(src, TRAIT_STUNIMMUNE)) || ignore_canknockdown)
 		if(absorb_stun(isnull(override_hardstun)? amount : override_hardstun, ignore_canknockdown))
 			return
+		//STAMINA COMBAT START
+		adjustStaminaLoss(isnull(override_stamdmg)? amount * 0.25 : override_stamdmg)
+		if(isnull(override_hardstun))
+			amount *= 0.01
```

Comment:
This is here only as a hack to force disarms and interrupt timers, it'd probably be better to make this directly force those things in the proc under the proper conditions (knockdown strength is more than 80 and the hardstun isnt being overridden)